release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3332,6 +3332,48 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.16.8
+[id="ocp-4-16-8_{context}"]
+=== RHSA-2024:5422 - {product-title} {product-version}.8 bug fix and security update
+
+Issued: 20 August 2024
+
+{product-title} release {product-version}.8, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:5422[RHSA-2024:5422] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:5425[RHBA-2024:5425] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.8 --pullspecs
+----
+
+[id="ocp-4-16-8-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when you clicked the {ols-official} link on the *Settings* page of your {product-title} cluster, the {ols} modal in Operator Hub did not open. With this update, the {ols} modal opens as expected. (link:https://issues.redhat.com/browse/OCPBUGS-38093[*OCPBUGS-38093*])
+
+* Previously, when you mirrored Operator catalogs with the `--rebuild-catalogs` argument, catalog cache was recreated on the local machine. This situation required extraction and use of the `opm` binary from the catalog image and this binary caused failure of either the mirroring operation or the catalog source on the cluster. These failures would happen because the supported operating system and platform of the `opm` binary casused a mismatch with the operating system and platform of `oc-mirror`. With this release, the value of `true` is applied to the `--rebuild-catalogs` argument by default so that any catalog rebuilds do not re-create internal cache. Additionally, this release updates the image from `opm serve /configs --cache-dir=/tmp/cache` to `opm serve /configs` so that creation of cache happens at pod startup, but note that this operation might cause longer pod startup operations. (link:https://issues.redhat.com/browse/OCPBUGS-38035[*OCPBUGS-38035*])
```

Comment:
```suggestion
* Previously, when you mirrored Operator catalogs with the `--rebuild-catalogs` argument, catalog cache was recreated on the local machine. This required extraction and use of the `opm` binary from the catalog image, which caused failure of either the mirroring operation or the catalog source. These failures would happen because the supported operating system and platform of the `opm` binary caused a mismatch with the operating system and platform of `oc-mirror`. With this release, the value of `true` is applied to the `--rebuild-catalogs` argument by default so that any catalog rebuilds do not re-create internal cache. Additionally, this release updates the image from `opm serve /configs --cache-dir=/tmp/cache` to `opm serve /configs` so that creation of cache happens at pod startup. Cache at startup might cause longer pod startup. (link:https://issues.redhat.com/browse/OCPBUGS-38035[*OCPBUGS-38035*])
```

Reply:
Was trying to trim this bug a little.

Reply:
:heavy_check_mark: